test/tests.json

```diff
@@ -11,6 +11,9 @@
     ["test@nominet.org.uk", "valid"],
     ["test@about.museum", "valid"],
     ["a@iana.org", "valid"],
+    ["êjness@iana.com", "valid"],
+    ["ñoñó1234@iana.com", "valid"],
+    ["伊昭傑@郵件.商務", "valid"],
```

Comment:
Hm, gotcha.

This is a good problem to notice. This also indicates that unicode support in the domain names will break `checkDNS`, as the `resolve` family of functions are not unicode compatible or aware, which goes back to my question about punycode. It seems we'll need just a little more to avoid breaking existing functionality in unexpected ways.